functions.php

```diff
@@ -215,15 +218,17 @@ function add_section_markup_before( $content, $section ) {
 		$style_attrs .= 'color: '. $text_color_custom .'; ';
 	}
 
+	$title = ! empty( $title ) ? ' data-section-link-title="' . $title . '"' : '';
+
 	ob_start();
 ?>
-	<section class="<?php echo $section_classes; ?>" style="<?php echo $style_attrs; ?>">
+	<section id="<?php echo $section->post_name; ?>" class="<?php echo $section_classes; ?>" style="<?php echo $style_attrs; ?>"<?php echo $title; ?>>
```

Comment:
We may run into issues with the section ID if a unique section is included on the page more than once--for example, the icon link callouts on the Admissions page--since ID attributes must be unique.  It would probably need to be updated in the section plugin instead of here, but having the ability to make the section ID optional would help.